bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/NanoleafHandlerFactory.java

```diff
@@ -107,11 +108,12 @@ private synchronized void registerDiscoveryService(NanoleafControllerHandler bri
         logger.debug("Discovery service for panels registered.");
     }
 
+    @SuppressWarnings("null")
     private synchronized void unregisterDiscoveryService(Thing thing) {
-        ServiceRegistration<?> serviceReg = discoveryServiceRegs.remove(thing.getUID());
-        if (serviceReg != null) {
+        @Nullable  ServiceRegistration<?> serviceReg = discoveryServiceRegs.remove(thing.getUID());
```

Comment:
Formatting seems a little bit wrong here, please reformat using eclipse or the mvn spotless plugin

Reply:
Eg:  `mvn spotless:apply -pl :org.openhab.binding.nanoleaf`

Reply:
applied spotless